ikarus/finiteelements/mechanics/materials/vanishingstress.hh

```diff
@@ -178,7 +178,7 @@ private:
     };
 
     auto Er    = E(fixedDiagonalVoigtIndices, fixedDiagonalVoigtIndices).eval().template cast<ScalarType>();
-    auto nonOp = Ikarus::NonLinearOperator(functions(f, df), parameter(Er));
+    auto nonOp = Ikarus::NonLinearOperator(Impl::noAssembler(), functions(f, df), parameter(Er));
```

Comment:
Very proud of this code🥇 
We have to write a CTAD + constructor where it defaults to noassembler if none is given, but I didn't really knew how to do it